src/Chapter2.hs

```diff
@@ -621,8 +640,10 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
-
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven (x:xs:xss) = x : takeEven xss
```

Comment:
Great pattern-matching skills 👏🏼 

Note, that you can use wildcard here, as you don't really use `xs`. Besides, `xs` is usually used for naming lists, but in this case, it is just an element 🙂 

```suggestion
takeEven (x:_:xs) = x : takeEven xs
```